src/tokens/Datatoken.ts

```diff
@@ -785,8 +785,15 @@ export class Datatoken {
       new this.web3.eth.Contract(this.datatokensAbi, dtAddress),
       this.config
     )
-    if ((await this.getDTPermissions(dtAddress, address)).paymentManager !== true) {
-      throw new Error(`Caller is not Fee Manager`)
+    const isPaymentManager =
+      (await this.getDTPermissions(dtAddress, address)).paymentManager === true
```

Comment:
if `paymentManager ` is already a bool why do we need the extra `===true` check ?